packages/fast-color-explorer/app/control-pane.tsx

```diff
@@ -0,0 +1,349 @@
+    setAccentBaseColor: (value: ColorRGBA64) => any;
+}
+
+export interface ControlPaneState {
+    accentColorBase: string;
+    neutralColorBase: string;
+}
+
+const accentShortcuts: string[] = values(AccentColors);
+
+const styles: any = (
+    designSystem: ColorsDesignSystem
+): ComponentStyleSheet<any, ColorsDesignSystem> => {
+    return {
+        "@global": {
+            body: {
+                fontFamily: '"Segoe UI", Arial, sans-serif',
+                fontSize: "14px",
+            },
+            ".sketch-picker": {
+                display: "flex",
+                flexDirection: "column",
+                boxSizing: "border-box !important",
+                width: "100% !important",
+                background: "transparent !important",
+                boxShadow: "none !important",
+                "& > div": {
+                    "&:nth-child(1)": {
+                        order: "2",
+                    },
+                    "&:nth-child(2)": {
+                        display: "none !important",
+                    },
+                    "&:nth-child(3)": {
+                        order: "4",
+                        paddingBottom: "4px",
+                    },
+                    "&:nth-child(4)": {
+                        borderTop: "none !important",
+                        paddingTop: "0 !important",
+                        "& div": {
+                            width: "24px !important",
+                            height: "24px !important",
+                            "& div": {
+                                borderRadius: format(
+                                    "{0} !important",
+                                    toPx(cornerRadius)
+                                )(designSystem),
+                            },
+                        },
+                    },
+                },
+            },
+            ".sketch-picker input": {
+                boxShadow: "none !important",
+                background: neutralFillInputRest(designSystem),
+                color: neutralForegroundRest(designSystem),
+                border: `1px solid ${neutralOutlineRest(designSystem)} !important`,
+                height: height()(designSystem),
+                fontSize: "14px !important",
+                borderRadius: format("{0} !important", toPx(cornerRadius))(designSystem),
+                paddingTop: "0 !important",
+                paddingBottom: "0 !important",
+                "&:focus": {
+                    outline: "none",
+                    boxShadow: `0 0 0 2px ${neutralFocus(designSystem)} inset !important`,
+                },
+            },
+        },
+        controlPane: {
+            position: "relative",
+            zIndex: "1",
+            padding: "12px",
+            boxSizing: "border-box",
+            color: neutralForegroundRest,
+            height: "100%",
+            maxWidth: "300px",
+            boxShadow: (config: ColorsDesignSystem): string => {

```

Comment:
This could be applyElevation now.

Reply:
I'll clean up in another PR - just trying to migrate.